server/plugins/builder/pages/summary.js

```diff
@@ -93,7 +94,9 @@ class SummaryViewModel {
 class SummaryPage extends Page {
   makeGetRouteHandler (getState) {
     return async (request, h) => {
+      this.langFromRequest(request)
```

Comment:
redundant?

Reply:
`langFromRequest` sets the language in the cookie also